invenio_rdm_records/theme/assets/scss/invenio_rdm_records/theme.scss

```diff
@@ -0,0 +1,154 @@
+/*
+ *   Copyright (C) 2020 CERN.
+ *   Copyright (C) 2020 Northwestern University.
+ *
+ * Invenio RDM Records is free software; you can redistribute it and/or modify
+ * it under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+@import "variables";
+
+// Preview
+.iiif-preview-img {
```

Comment:
minor: Could move

```css
.preview-iframe {
  border-style: none;
}
```

from invenio-app-rdm here too.

Reply:
True! Thanks for spotting!